pkg/occlient/utils.go

```diff
@@ -0,0 +1,23 @@
+package occlient
+
+import scv1beta1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+
+func isHidden(tags []string) bool {
```

Comment:
I'd actually make this function a little more generic: `hasTag(tags []string, tag string)` instead of making it specific to `hidden`. Asking selfishly because I actually need such a function ;)

Reply:
ack